src/index.html

```diff
@@ -9,7 +9,7 @@
 
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
         <!-- Place favicon.ico in the root directory -->
-
+        <link rel="stylesheet" href="octicons/octicons.css">
```

Comment:
This would be better included as a link, perhaps like this: http://cdnjs.com/libraries/octicons